_posts/2019-03-22-cockpit-190.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cockpit 190
+author: kkoukiou
+date: 2019-03-22
+tags: cockpit linux
+slug: cockpit-190
+category: release
+summary: Cockpit with new features in logs and machines
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 190.
+
+### Logs: Filter log entries by service
+
+![Filtering logs by service](/images/logs-filter-service-name.png)
+
+Logs page allows now to filter the logs results by a specific service.
```

Comment:
"The Logs page now allows you to only view the logs for a specific service." It's basically repeating the heading, but it looks bad without any text.

Reply:
Done